source/agora/consensus/Genesis.d

```diff
@@ -46,23 +46,55 @@ unittest
 
 ///
 private immutable Hash GenesisMerkleRoot =
-    Hash(`0xd04f8dd627ec5245129527b2327ad73d1157d483441a249d7eecd086ee1c4775`
-         ~ `3ad14d410f2bf128d8737366038c85d223e5701b73c7238089f52898ecd0e1b2`);
+    Hash(`0xdb6e67f59fe0b30676037e4970705df8287f0de38298dcc09e50a8e85413959ca`
+         ~ `4c52a9fa1edbe6a47cbb6b5e9b2a19b4d0877cc1f5955a7166fe6884eecd2c3`);
 
 /// The single transaction that are part of the genesis block
 public immutable Transaction GenesisTransaction =
 {
     TxType.Payment,
     inputs: [ Input.init ],
     outputs: [
-        Output(Amount(40_000_000 / Block.TxsInBlock), GenesisOutputAddress),
-        Output(Amount(40_000_000 / Block.TxsInBlock), GenesisOutputAddress),
-        Output(Amount(40_000_000 / Block.TxsInBlock), GenesisOutputAddress),
-        Output(Amount(40_000_000 / Block.TxsInBlock), GenesisOutputAddress),
-        Output(Amount(40_000_000 / Block.TxsInBlock), GenesisOutputAddress),
-        Output(Amount(40_000_000 / Block.TxsInBlock), GenesisOutputAddress),
-        Output(Amount(40_000_000 / Block.TxsInBlock), GenesisOutputAddress),
-        Output(Amount(40_000_000 / Block.TxsInBlock), GenesisOutputAddress),
+        Output(
+            Amount(
+                Amount.MaxCoinSupply * Amount.Unit / Block.TxsInBlock
```

Comment:
@Geod24 there is already a `MaxUnitSupply` that we could use here, but we can't divide an `Amount`. Because I wanted to originally suggest here:

```D
Amount.MaxUnitSupply / Block.TxsInBlock
```

So what should we do here? Because we can't access `.value` to divide the amount (it's private).

Reply:
Amount.value is private

Reply:
@Geod24 how about my question here? `Amount.MaxUnitSupply` already exists, but it's an `Amount`, which is not divisible.